Application/EdFi.Ods.SandboxAdmin/Models/Results/AdminActionResult.cs

```diff
@@ -23,10 +23,12 @@ public bool HasMessage
             get { return !string.IsNullOrEmpty(Message); }
         }
 
-        public string[] FailingFields
-        {
-            get { return _failingFields.ToArray(); }
-        }
+        // SF: I agree with this recommendation from SonarSource. However, I am nervous
+        // that this isn't being falgged as unused, yet changing to a method isn't
+        // breaking anything. Would need to test this thoroughly. Might prefer to
+        // make an exception and leave this alone. For more information, see 
+        // https://rules.sonarsource.com/csharp/RSPEC-2365
+        public string[] GetFailingFields() => _failingFields.ToArray();
```

Comment:
Thoughts on this one?

Reply:
used in javascript, so method may not be an option here

Reply:
OK, I'll reverse that. Not that this will be merged - but I don't want anyone to copy and paste something inappropriate.